### DISCUSSION TITLE
HDR10+ in X265

### DISCUSSION THREAD

**Discussion by papa-pitufo** [staxrip/staxrip#776] [General]
Hi everyone.

Recently i read a mediaInfo with this
Format : HEVC
Format/Info : High Efficiency Video Coding
Format profile : Main 10@L5.1@High
**HDR format : SMPTE ST 2094 App 4, Version 1, HDR10+ Profile B compatible**
Codec ID : V_MPEGH/ISO/HEVC

Does anyone knows if Patman is going to introduce this Transfer Matrix in next release of X265?
How would it be used?
Do you have to use HDR10 json file? yes or yes.

Thanks in advantage.

**Comment by Dendraspis** [comment]
What shall Patman introduce?
You have to extract the metadata into a json file, that you can pass via parameter to x265. This works for a very long time - no need to introduce something in/for x265.

**Comment by papa-pitufo** [comment]
Sorry, I explain very bad

First, I can't do any video file HDR10 or HDR10+ because I didn't find any BD disc 4k that I could extract json metadata.  And I did many mkv 4k.  Probably I have bad luck.

This is theorically:

If you want to use HDR10+ you need json file metadata fot HDR10 and select SMTPE ST 2094 for HDR10+.
If you use SMTPE ST 2084, you use HDR or if you can need json also, then you use HDR10.

in other words
HDR: SMTPE 2084
HDR10: HDR + json file
HDR10+: json file + SMTPE 2094

Or am I wrong?, in which case, please clarify it for me.


Many thanks in advantage
